scripts/chroot.sh

```diff
@@ -22,11 +22,6 @@ apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 1EBD81D9
 apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 91E7EE5E
 wget -qO - https://download.jitsi.org/jitsi-key.gpg.key | apt-key add -
 
-if [ ${arch} == 'x86_64' ];   # 64-bit
```

Comment:
that should never be executed. How does it solve the problem of the black screen?

Reply:
yes, this never get executed.
But the breaking of the ISO starts from here.